src/main/java/com/dife/api/config/AWSConfig.java

```diff
@@ -15,13 +15,13 @@
 @Profile("!test")
 public class AWSConfig {
 
-	@Value("${spring.aws.access-key:#{null}")
+	@Value("${spring.aws.access-key}")
```

Comment:
확인해봤는데요! 제가 끝에 `}`를 추가하지 않아서 에러가 나서 그렇네요!
시간나시면 해당 PR에서 아래와 같이 변경해주시면 감사하겠습니다 :) 
```suggestion
@Value("${spring.aws.access-key:#{null}}")
```

Reply:
```
@Value("${spring.aws.secret-key}")
private String secretKey;

@Value("${spring.aws.session-token}")
private String sessionToken;
```

이 두 부분도 같은 방식으로 부탁드려요! :bow: